experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -65,12 +65,17 @@ import {
   serviceInstanceIdDetectorSync,
 } from '@opentelemetry/resources';
 import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
-import { logs } from '@opentelemetry/api-logs';
+import { logs, NoopLoggerProvider } from '@opentelemetry/api-logs';
```

Comment:
I think we did not intend to export `NoopLoggerProvider` from the logs API. This export will disappear in the future. Ideally we'd use something else to determine if the `LoggerProvider` has been registered (maybe spying on the API?)